src/arrayMethodSort.js

```diff
@@ -3,9 +3,42 @@
 /**
  * Implement method Sort
  */
+
+const compareAsStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  }
```

Comment:
```suggestion
const compareAsStrings = (a, b) => {
  if (String(a) > String(b)) {
    return 1;
  }
```

end you don't need create vars for this